atc/worker/worker.go

```diff
@@ -371,64 +387,55 @@ func (worker *gardenWorker) createVolumes(logger lager.Logger, isPrivileged bool
 
 	inputDestinationPaths := make(map[string]bool)
 
-	for _, inputSource := range spec.Inputs {
-		var inputVolume Volume
+	localInputs := make([]mountableLocalInput, 5)
+	nonlocalInputs := make([]mountableRemoteInput, 5)
```

Comment:
Is there a reason to have a limit of 5 local/non-local inputs? for large tasks with lots of `get` steps before it, we could go over 5 inputs.

Reply:
nope but it needs a intial value or nothing at all. Everytime `append` sees a full array, it reinitializes it so I was just making a good guess that most pipelines will have less than 5 inputs. Avoiding atleast 1 realloc when append is called for the first time.